### ISSUE TITLE
Reconsider use of fallback date?

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/pdp#135]
Currently, all apps except PCDS with date filtering use a fallback date (typically the last date available in the dataset's time system) when the input date is invalid. With our current UI, lacking a datepicker (due to multiple calendars), it is easier to enter an invalid date.

All apps except PCDS use hyperlinks that are dynamically updated for data download. The PCDS app uses buttons. In the PCDS app, the buttons are disabled when an invalid date has been entered.

Question: Would it be better to "disable" the data download hyperlinks when an invalid date is entered? What would the disabled state be? Possibilities:

- Remove hyperlink altogether
- Make it not be a hyperlink (just the label text), with different formatting to show it is disabled.

**Comment by jameshiebert** [comment]
Yeah, this behaviour was one of the first things that I honed in on out of curiosity. I think that the current behaviour of using defaults is *fine*, but we should let the user know that that's what's happening in the red text. A brief, "Defaulting to 1950/01/01" or whatever, should be sufficient.

The other thing, is that the error text doesn't follow the UI principle of helping the user *correct* the error. OK March 31st isn't valid for a 360 day calendar. What *is* valid? We know that, but many users will not. Is it worth pointing people to docs on the various calendars somehow?
